README.md

```diff
@@ -1,21 +1,82 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
 
-## Use the client-side tool
+## scalar-admin client tool
+
+The scalar-admin client tool is used to manage the admin interface implemented applications.
+It will help to `PAUSE` and `UNPAUSE` the application without losing the transaction.
+
+### Usage
+
+You can manage the scalar-admin integrated applications in the following ways
```

Comment:
Isn't it weird to go to `install` after `usage`?

Follow this.

* Install
  * Fat jar
  * Docker container
  * Build from sources
* Run
  * Fat jar
  * Docker container

Reply:
Thank you for your suggestion.
Document updated with the above suggestion.